src/models/Component.js

```diff
@@ -103,6 +103,37 @@ class Component extends FileInformation {
     }
   }
 
+  /**
+   * Create all parents attributes from definitions.
+   * @param {ComponentAttributeDefinition[]} definitions - All parents attributes order from most
+   * depth to lower depth.
```

Comment:
from most depth to lower depth -> from greatest to least depth.

Reply:
done